src/main/daml/Daml/Finance/Interface/Lifecycle/Event.daml

```diff
@@ -3,7 +3,8 @@
 
 module Daml.Finance.Interface.Lifecycle.Event where
 
--- | Type synonym for `Event`.
```

Comment:
I would maintain this description of `type I` as it is printed in the docs.

Reply:
Whoops, I've added it back.